util/check-format-commit.sh

```diff
@@ -82,19 +67,19 @@ git show $COMMIT | awk -v mycmt=$COMMIT '
         printf mycmt " " myfile " " $3 "\n"
     }' >> $TEMPDIR/ranges.txt || true
 
-# filter out anything that matches on a filter regex
-for i in ${EXCLUDED_FILE_REGEX[@]}
+# filter in anything that matches on a filter regex
+for i in ${FILE_ALLOWLIST[@]}
 do
     touch $TEMPDIR/ranges.filter
-    grep -v "$i" $TEMPDIR/ranges.txt >> $TEMPDIR/ranges.filter || true
-    REMAINING_FILES=$(wc -l $TEMPDIR/ranges.filter | awk '{print $1}')
-    if [ $REMAINING_FILES -eq 0 ]
-    then
-        echo "This commit has no files that require checking"
-        exit 0
-    fi
-    mv $TEMPDIR/ranges.filter $TEMPDIR/ranges.txt
+    grep "$i" $TEMPDIR/ranges.txt >> $TEMPDIR/ranges.filter || true
```

Comment:
to be clear, space in this line, with an explanatory comment above
```suggestion
    grep "$i " $TEMPDIR/ranges.txt >> $TEMPDIR/ranges.filter || true
```